WooCommerce/Classes/ViewRelated/Coupons/CouponDetails/CouponDetails.swift

```diff
@@ -166,6 +167,13 @@ struct CouponDetails: View {
             }
             .background(Color(.listBackground))
             .ignoresSafeArea(.container, edges: [.horizontal, .bottom])
+            .sheet(isPresented: $showingEnableAnalytics) {
+                EnableAnalyticsView(viewModel: .init(siteID: viewModel.siteID),
+                                    presentingController: noticePresenter.presentingViewController,
```

Comment:
This looks a bit like a hack; we can send the hosting controller to `CouponDetails` and use it here instead, but I'm not sure I like it either.